### PR TITLE
add souravsk

### PR DESCRIPTION
**Pull request by souravsk**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1163"></a>

### CODE REVIEW

File: public/data/souravsk.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      "title": "Learning DSA",
+      "date": "January 2022",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Learning"
+    }
+    {
+      "title": "Learning Kubernetes",
+      "date": "January 2022",
+      "icon": "dollar",
```
Comment (krshdev) [inline public/data/souravsk.json:34]: Hey, the icon would be shown as a $ are you sure about that? 😅

**Review by ssensalo — COMMENTED** [review]
Looks good, just clarify with what @krshdev has identified

File: public/data/souravsk.json (hunk 1 above)
Comment (souravsk) [reply public/data/souravsk.json:34]: Sorry I didn't see that I will remove that 
After remove i have to create a new pull request

**Review by Panquesito7 — APPROVED** [review]
Looks good 👍 apart from the above. Let us know if you need any help. 😃

**Comment by souravsk** [comment]
> Looks good 👍 apart from the above. Let us know if you need any help. 😃
> 
After I made changes do i have to create a new pull request. Or If I can commit and push in this same pull request.
If yes than please let me know how

**Comment by schmelto** [comment]
> > Looks good 👍 apart from the above. Let us know if you need any help. 😃
> > 
> After I made changes do i have to create a new pull request. Or If I can commit and push in this same pull request.
> If yes than please let me know how 

You can commit those changes into the same forked branch this will sync the changes in this or

File: public/data/souravsk.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/souravsk",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/sourav_sovu_",
+      "icon": "twitter"
+    }
```
Comment (krshdev) [inline public/data/souravsk.json:16]:
```suggestion
    },
```

File: public/data/souravsk.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  ],
+  "milestones": [
+    {
+      "title": "Learning DSA",
+      "date": "January 2022",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Learning"
+    }
```
Comment (krshdev) [inline public/data/souravsk.json:30]:
```suggestion
    },
```

**Review by krshdev — CHANGES_REQUESTED** [review]
Hey, just one more thing after the dollar sign, the failing build

File: public/data/souravsk.json (hunk 3 above)
Comment (souravsk) [reply public/data/souravsk.json]: i made all the change

**Review by ssensalo — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**